docs/source/upgrade.rst

```diff
@@ -0,0 +1,42 @@
+Overview
+========
+
+If you’re familiar with previous releases of Hyperledger Fabric, you’re aware
+that upgrading the nodes and channels to the latest version of Fabric is, at a
+high level, a four step process.
+
+1. Backup the ledger and MSPs.
+2. Upgrade the orderer binaries in a rolling fashion to the latest Fabric version.
+3. Upgrade the peer binaries in a rolling fashion to the latest Fabric version.
+4. Update the orderer system channel and any application channels to the latest
+   capability levels, where available. Note that some releases will have
+   capabilities in all groups while other releases may have few or even no new
+   capabilities at all.
+
+For more information about capabilities, check out :doc:`capabilities_concept`.
+
+For a look at how these upgrade processes are accomplished, please consult these
+tutorials:
+
+1. :doc:`upgrading_your_components`. Components should be upgraded to the latest
+   version before updating any capabilities.
+2. :doc:`updating_capabilities`. Completed after updating the versions of all nodes.
+3. :doc:`enable_cc_lifecycle`. Necessary to add organization specific endorsement
+   policies central to the new chaincode lifecycle for Fabric v2.0.
+
+As the upgrading of nodes and increasing the capability levels of channels is by
+now considered a standard Fabric process, we will not show the specific commands
```

Comment:
One question we see alot is whether the SDK versions need to match the Fabric versions. It would be nice if we could address that in this doc somewhere.

Reply:
Added a statement here from Bret about SDK compatibility. Basically, it depends from release to release and feature to feature. Best practice is to always be at the latest version.